vibi-dpu/src/bitbucket/auth.rs

```diff
@@ -49,13 +50,13 @@ pub async fn get_access_token_from_bitbucket(code: &str) -> Option<BitbucketAuth
     return Some(response_json);
 }
 
-pub async fn refresh_git_auth(clone_url: &str, directory: &str) -> Option<String>{
+pub async fn refresh_git_auth(review: &Review) -> Option<String>{
```

Comment:
Semantically, I am not able to understand why a "review" object would be required to refresh auth. Are we sure that we are not misappropriating a struct to do more than it is supposed to do?

Reply:
It is required since data fields inside it need to be updated and saved to the db in case git auth actually needs to be refreshed